roles/lib_openshift/src/test/unit/conftest.py

```diff
@@ -0,0 +1,55 @@
+import inspect
+
+import pytest
+
+from lib_openshift.library import *  # noqa: F401,F403
```

Comment:
I agree. ```*``` imports make me worry. If we truly don't know what is being imported then I agree with generating ``__all__`` as @rhcarvalho noted. If we do know what we are importing I'd rather see this explicitly import.

As a side note unittests generally should have these attributes:

- Fast
- Reliable
- Easily readable

I'm a bit worried that things such as [this clever bit of code](https://github.com/detiber/openshift-ansible/blob/32f826c860ef6bb0cf692053e59b607b8fcd66dc/roles/lib_openshift/src/test/unit/conftest.py#L12-L19) could make it hard for someone to debug why a test has failed. But I have not done a review on all of the changes so I I could be off base.